File: 2.0/java/texts/findTextAutoReplys.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.TextAutoReply;
import com.callfire.api.client.api.campaigns.model.request.FindTextAutoRepliesRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        FindTextAutoRepliesRequest request = FindTextAutoRepliesRequest.create()
            .number("12132000384")
            .offset(0L)
            .limit(10L)
            .fields("items(id,match,keyword,message)")
            .build();
        Page<TextAutoReply> textAutoReplies = client.textAutoRepliesApi().find(request);
    }
}

File: 2.0/csharp/texts/findTextAutoReplys.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Campaigns.Model.Request;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindTextAutoRepliesRequest
        {
            Number = "12132000384",
            Offset = 0,
            Limit = 10,
            Fields = "items(id,match,keyword,message)"
        };
        Page<TextAutoReply> autoReplies = client.TextAutoRepliesApi.Find(request);
    }
}
